src/App/Repository/PostRepository.php

```diff
@@ -389,6 +389,23 @@ public function setSearchParams(SearchData $search)
             $this->search_query->offset(intval($sorting['offset']));
         }
 
+        if ($search->getFilter('limitPosts')) {
+            // the post management permission is associated with bigger limits
+            // we would like to do this with API scopes in the future
+            $isUnprivileged = !$this->postPermissions->canUserManagePosts($this->getUser());
+            $requestedLimit = $sorting['limit'] ?? 0;
+            if ($isUnprivileged) {
+                $unprivilegedLimit = env('API_POST_LIST_MAX_LIMIT', config('posts.list_max_limit'));
```

Comment:
the convention in Laravel/Lumen afaik is to put the env lookup in the config file, i.e. , as you can see in config/database.php:

```
'mysql' => [
            'host'      => env('DB_HOST', 'localhost'),
            'read' => [
...
```

could you please use that convention too?

Reply:
will do when I get a chance

Reply:
@tuxpiper I made this change

Reply:
the tests are failing during the env setup stage with: Could not open input file: composer-setup.php
I tried merging master using the button on here and it is still the same error. I don't think that's related to my changes 🤷